WordPress/src/main/java/org/wordpress/android/ui/TextInputDialogFragment.java

```diff
@@ -51,6 +51,7 @@ public static TextInputDialogFragment newInstance(String title,
         return textInputDialogFragment;
     }
 
+    /** @noinspection checkstyle:RightCurly*/
```

Comment:
What was the reason behind adding this suppression?

Reply:
I'm sorry, please forgive my carelessness. It's useless.